src/kimmdy/kimmdy-yaml-schema.json

```diff
@@ -194,6 +187,25 @@
       "type": "string",
       "pytype": "Path"
     },
+    "restart": {
+      "title": "restart",
+      "type": "object",
+      "description": "Restart from a previous run.",
+      "properties": {
+        "run_directory": {
+          "type": "string",
+          "pytype": "Path",
+          "description": "KIMMDY run directory to restart from"
+        },
+        "manual": {
```

Comment:
cool idea, but what do we use it for?

Reply:
yeah, Kai wanted some functionality and I think I misunderstood him. I will change it into starting at a certain point in the sequence if all the files for the steps previous to the restart step are in the restart directory. This way one can branch from a kimmdy run